service/storage_service.cc

```diff
@@ -11,6 +11,8 @@
 
 #include "storage_service.hh"
 #include "dht/boot_strapper.hh"
+#include <boost/range/adaptor/filtered.hpp>
+#include <boost/range/adaptor/transformed.hpp>
```

Comment:
Nit: I think those are no longer needed, no?

Reply:
Indeed, I checked and it's no longer needed. Will remove in the followup.